Extensions/Multiplayer/multiplayerobjectruntimebehavior.ts

```diff
@@ -92,9 +92,14 @@ namespace gdjs {
       // To handle this case and avoid having an object not synchronized, we set a timeout to destroy the object
       // if it has not been assigned a networkId after a short delay.
       this._destroyInstanceTimeoutId = setTimeout(() => {
-        if (!owner.networkId && gdjs.multiplayer.isLobbyGameRunning()) {
+        const sceneNetworkId = this.owner.getRuntimeScene().networkId;
```

Comment:
I wonder if in the future we'll need a real `playerNumber` or equivalent methods like `disableSynchronization` as for variables. So that we can differentiate a scene that is to be synchronized but is not yet reconciled (networkId is empty) from those which are not to be synchronized.

Reply:
probably yeah. playerNumber being 0 by default for scenes, but sync can be disabled. Or something like that.
Didn't have the request until now.... So maybe we're imagining a need?